
'a'.Split([object[]])
'Continued after error'
